runtime/gc_glue_java/ConfigurationDelegate.hpp

```diff
@@ -354,6 +354,11 @@ class MM_ConfigurationDelegate
 		return _maximumDefaultNumberOfGCThreads;
 	}
 
+	void setMaxGCThreadCount(MM_EnvironmentBase* env, uintptr_t maxGCThreads)
+	{
+		_maximumDefaultNumberOfGCThreads = maxGCThreads;
```

Comment:
should this not be min(_maximumDefaultNumberOfGCThreads, maxGCThreads) - we might lose the hard limit?

Reply:
after looking elsewhere and realizing this is intentional to reduce the limit and the useage of this API is temporary, this is ok as-is